code/espurna/config/custom.h.example

```diff
@@ -0,0 +1,95 @@
+// ------------------------------------------------------------------------------
+// Example file for custom.h 
+// Either copy and paste this file then rename removing the .example or create your
+// own file: 'custom.h'
+// This file allows users to create their own configurations.
+// See 'code/espurna/config/general.h' for default settings.
+//
+// See: https://github.com/xoseperez/espurna/wiki/Software-features#enabling-features
+// and 'code/platformio_override.ini.example' for more details.
+// ------------------------------------------------------------------------------
+
+
+// Example of a Sonoff Basic board with some options disabled to reduce firmware size.
+
+#if defined(MY_SONOFF_BUILD01)    // This matches the 'src_build_flags = -DMY_SONOFF_BUILD01'

```

Comment:
*eaasl, maybe I misunderstand what 'matches' means here*

"This section will be used when `src_build_flags` contains -DMY_SONOFF_BUILD01"?

Reply:
Yes, your version makes more sense. I was trying to say the the name in brackets `( MY_SONOFF_BUILD01)` has to be same as the `src_build_flags` name in platformio.override.ini